lib/index.js

```diff
@@ -309,4 +324,236 @@ module.exports = {
 
     return debugTree(new MergeTrees([ember, templateCompiler, jquery]), 'vendor:final');
   },
+
+  _issueGlobalsDeprecation() {
+    if (process.env.EMBER_ENV === 'production') {
+      return;
```

Comment:
Don't we still need to set the bootstrap string?

Reply:
There is a default (same as the current one) on the prototype (?)